apps/web/nuxt.config.ts

```diff
@@ -41,7 +42,10 @@ export default defineNuxtConfig({
   },
   pages: true,
   hooks: {
-    'build:before': async () => await fetchConfiguration(),
+    'build:before': async () => {
+      await fetchConfiguration();
+      generateScssVariables();
```

Comment:
```suggestion
      if (process.env.FETCH_REMOTE_CONFIG === '1') {
        await fetchConfiguration();
        generateScssVariables();
      } else {
        console.warn(`Fetching PWA settings is disabled! Set FETCH_REMOTE_CONFIG in .env file.`);
      }
```
**suggestion**: I would extract the remote config flag, so that it applies to both fetching and generating the colours. This way people can still create a completely custom palette if they want.

In this case we could remove the following from `fetchConfiguration`:
* line 11
* line 22
* lines 75-79